index.js

```diff
@@ -106,6 +106,43 @@ var Controllers = {
     };
   },
 
+  TabBarControllerIOS: function (id) {
+    return {
+      setHidden: function (params) {
+        return RCCManager.TabBarControllerIOS(id, "setTabBarHidden", params);
+      }
+    };
+  },
+
+  Modal: {
+    showLightBox: function(componentId, style) {
+      if (style.backgroundColor !== undefined) {
+        style.backgroundColor = processColor(style.backgroundColor);
+      }
+      RCCManager.modalShowLightBox(componentId, style);
+    },
+    dismissLightBox: function() {
+      RCCManager.modalDismissLightBox();
+    },
+    showController: function(appKey, animated) {
```

Comment:
Let's change the signature here to be like the new `setRootController` (see comment below).

`showController: function (appKey, animationType = 'none')`

This will allow us in the future to support multiple types of animations.